velox/exec/prefixsort/PrefixSortEncoder.h

```diff
@@ -76,10 +76,11 @@ class PrefixSortEncoder {
   FOLLY_ALWAYS_INLINE static std::optional<uint32_t> encodedSize(
       TypeKind typeKind,
       uint32_t maxStringPrefixLength,
-      bool columnHasNulls) {
+      bool columnHasNulls,
```

Comment:
How about consolidating to one parameter: `bool includeNullByte`?

Reply:
The caller does not need to know when the encoder need to include null byte. It is encoder's responsibility to decide it. I think current approach is more clear.